pkg/lua/Makefile.lua

```diff
@@ -1,9 +1,13 @@
 SRC := $(filter-out lua.c luac.c,$(wildcard *.c))
 
-# This builds using POSIX system calls and some extra libraries, and
+# This builds for native using POSIX system calls and some extra libraries, and
 # removes a compiler warning that warns against using tmpnam().
 ifeq ($(BOARD),native)
 CFLAGS += -DLUA_USE_LINUX
 endif
 
+# This redefines a lua prng function, avoiding the need for the _times system
+# call, which RIOT doesn't provide
+CFLAGS += -D'l_randomizePivot()=0'
```

Comment:
I think it's a cleaner approach to remove this function with the patch

Reply:
The lua source is written in such a way that one is supposed to redefine things to change them. In this case, the definition of `l_randomizePivot()` is wrapped inside a `#ifndef l_randomizePivot`

Reply:
I'm for putting all the patches for this fixup together in one patch. eg

`#define l_randomizePivot() 0`

just above `if !defined(l_randomizePivot)`

sorts it out.